kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -60,12 +60,24 @@
       + "are used.";
   public static final String ID_DEFAULT = "";
 
+  public static final String MAX_POLL_RECORDS_CONFIG = "max.poll.records";
+  private static final String MAX_POLL_RECORDS_DOC =
+          "The maximum number of records returned in a single call to poll().";
+  // ensures poll is frequently needed and called
+  public static final String MAX_POLL_RECORDS_VALUE = "10";
+
   public static final String HOST_NAME_CONFIG = "host.name";
   private static final String HOST_NAME_DOC =
       "The host name used to generate absolute URLs in responses. If empty, the default canonical"
       + " hostname is used";
   public static final String HOST_NAME_DEFAULT = "";
 
+  public static final String CONSUMER_MAX_THREADS_CONFIG = "consumer.threads";
+  private static final String CONSUMER_MAX_THREADS_DOC =
+      "The maximum number of threads to run consumer requests on."
+      + " The value of -1 denotes unbounded thread creation";
+  public static final String CONSUMER_MAX_THREADS_DEFAULT = "1";
```

Comment:
Is this right? We document 100 above.

Reply:
Sounds fair considering it's a cached pool

Reply:
Isn't 16 a bit low? Given that this is a cached thread pool, should we be considering a larger value by default? Since we decrease the thread pool size if there are idle threads, it seems like the risk is low. As @stanislavkozlovski said, the main downside is there were a lot of 1 group consumers, but not sure how common that is.

Reply:
Also, maybe the core number should be higher than 0 so that we don't end up with no threads at all during idle periods.